source/_components/switch.recswitch.markdown

```diff
@@ -0,0 +1,48 @@
+---
+layout: page
+title: "Ankuoo Rec Switch"
+logo: ankuoo_recswitch.png
+description: "home-assistant.io web presence"
+date: 2018-10-05 08:00
+sidebar: true
+comments: false
+sharing: true
+footer: true
+ha_release: "0.80"
```

Comment:
"0.80" has already been cut off as a beta. Please update this to the next major release (0.81).